src/odoc/env.ml

```diff
@@ -35,7 +37,7 @@ module Accessible_paths = struct
 
   let find_file_by_name t name =
     let uname = name ^ ".odoc" in
-    let lname = String.uncapitalize_ascii name ^ ".odoc" in
+    let lname = String.uncapitalize name ^ ".odoc" in
```

Comment:
Total nit, but I recommend calling these functions in `Compat.String` by their 4.03 names (`uncapitalize_ascii`, etc.), so we can later just delete that module and `open Compat`, and not have to edit the rest of the code.

Reply:
Makes sense!